dev-docs/RFCs/v7.0/google-maps-integration.md

```diff
@@ -103,15 +100,24 @@ Supported features:
 
 - Layers
 - Effects
-- Auto-highliting
+- Auto-highlighting
 - Transitions / Animations
-- Interaction callbacks
+- `onHover` and `onClick` callbacks
 
 Not supported features:
 
 - Multi view
 - Controller
 - React integration (there is no official React wrapper of Google Maps)
+- Gesture event callbacks (e.g. `onDrag*`)
+
+### Support tilting?
+
+According to official docs:
+
+> Controls the automatic switching behavior for the angle of incidence of the map. The only allowed values are 0 and 45. The value 0 causes the map to always use a 0° overhead view regardless of the zoom level and viewport. The value 45 causes the tilt angle to automatically switch to 45 whenever 45° imagery is available for the current zoom level and viewport, and switch back to 0 whenever 45° imagery is not available (this is the default behavior). 45° imagery is only available for satellite and hybrid map types, within some locations, and at some zoom levels.
+
+There is no obvious way to get the fractional tilting of the base map. We will hide the deck.gl layers when tilt is larger than 0.
```

Comment:
The tilt is never fractional – it's either 0º or 45º, and nothing in between. See [this example](https://developers.google.com/maps/documentation/javascript/examples/aerial-rotation) and try zooming out once. It's [possible to get the tilt](https://stackoverflow.com/questions/37313360/google-maps-javascript-api-v3-how-do-i-make-gettilt-method-return-current-ti) by listening for a `tilt_changed` event. However, I'm not confident the actual imagery is _exactly_ at 45º, and would be surprised if it lines up easily and consistently.